docs/go/how-to-set-startworkflowoptions-in-go.md

```diff
@@ -250,8 +250,10 @@ if err != nil {
 
 ### `SearchAttributes`
 
+Values of custom Search Attributes which must be [added](TODO:link to add search attributes here) to cluster first. 
```

Comment:
How?

Reply:
IMO, a link to "how to add custom Search Attributes using tctl" would be included in the RelatedReadList at the bottom of the section.